src/app/core/services/config/config.service.ts

```diff
@@ -189,11 +194,14 @@ export class ConfigService {
       .then(() => this.appConfig.setUTCOffset(utcOffset))
       .then(() => this.appConfig.setPrevUTCOffset(prevUtcOffset))
       .then(() => this.regenerateSchedule())
+      .then(() => this.appServerService.updateSubjectTimezone())
```

Comment:
Also need to do the following-

- Update the language
- Update the status of notification to delivered if the app was opened by clicking on notification
-  update the lastOpened value of the user to now.